### PR TITLE
fix: absolute rootDir on Windows

### PR DESCRIPTION
**Pull request by semoal**
It's a bit workaround, I' have to look deeper to analyze the issue, but meanwhile it works on Windows and Mac as usual 👍🏻

### CODE REVIEW

File: examples/next-js/locale/cs/messages.po [hunk 1]
```diff
@@ -14,46 +14,46 @@ msgstr ""
 "Plural-Forms: \n"
 
 #: pages/index.tsx:38
-msgid "Checkout LinguiJS examples and usecases."
-msgstr "Podívej se na ukázky a příklady LinguiJS"
+#~ msgid "Checkout LinguiJS examples and usecases."
```
Comment (tricoder42) [inline examples/next-js/locale/cs/messages.po:17]: It seems like it doesn't find any messages as they're all marked as obsolete.

**Review by tricoder42 — COMMENTED** [review]
Thank you for working on this @semoal, but the fix doesn't seem to work. No messages were extracted, so it looks like the path isn't correct.

**Comment by semoal** [comment]
> Thank you for working on this @semoal, but the fix doesn't seem to work. No messages were extracted, so it looks like the path isn't correct.

Ugh i take a further look then

**Comment by semoal** [comment]
Finally fixed, was an issue with glob library that says:
`
Results from absolute patterns such as /foo/* are mounted onto the root setting using path.join. On windows, this will by default result in /foo/* matching C:\foo\bar.txt.
`

So on Mac was working properly but on windows was resolving to root of the system..

**Comment by semoal** [comment]
With this fixed, we'ready to ship :shipit:

**Review by tricoder42 — APPROVED** [review]
Amazing work! Thanks for that 👍

**Pull request merged**